code/game/objects/structures/hybrisa_props.dm

```diff
@@ -0,0 +1,2359 @@
+	icon = 'icons/obj/structures/prop/urban/vehiclesexpanded.dmi'
+	icon_state = "MeridianCar_1"
+
+
+// Very Large Vehicles
+
+/obj/structure/prop/urban/vehicles/large_vehicles
+	icon = 'icons/obj/structures/prop/urban/128x32_vehiclesexpanded.dmi'
+	icon_state = "armoredtruck_nt_security_1"
+	density = TRUE
+	layer = ABOVE_MOB_LAYER
+
+// Misc
+/obj/structure/prop/urban/vehicles/large_vehicles/ambulance
+	name = "ambulance"
+	desc = "Seems to be broken down."
+	icon_state = "ambulance"
+	bound_height = 64
+	bound_width = 96
+
+// Armored Truck
+
+/obj/structure/prop/urban/vehicles/large_vehicles/armored_trucks
+	icon_state = "armoredtruck_wy_security_1"
+	bound_height = 64
+	bound_width = 96
+/obj/structure/prop/urban/vehicles/large_vehicles/armored_trucks/nt_security/truck_1
+	name = "Nanotrasen security truck"
+	desc = "Seems to be broken down."
+	icon_state = "armoredtruck_nt_security_1"
+
+/obj/structure/prop/urban/vehicles/large_vehicles/armored_trucks/nt_security/truck_2
+	name = "Nanotrasen security truck"
+	desc = "Seems to be broken down."
+	icon_state = "armoredtruck_nt_security_2"
+
+// Armored Truck Goods
+/obj/structure/prop/urban/vehicles/large_vehicles/armored_trucks/heavy_loader/white
+	name = "heavy loader truck"
+	desc = "Seems to be broken down."
+	icon_state = "armoredtruck_white_white"
+
+/obj/structure/prop/urban/vehicles/large_vehicles/armored_trucks/heavy_loader/white_teal
+	name = "heavy loader truck"
+	desc = "Seems to be broken down."
+	icon_state = "armoredtruck_white_teal"
+
+/obj/structure/prop/urban/vehicles/large_vehicles/armored_trucks/heavy_loader/blue_white
+	name = "heavy loader truck"
+	desc = "Seems to be broken down."
+	icon_state = "armoredtruck_blue_white"
+
+// Mega-Hauler Trucks 128x64
+
+/obj/structure/prop/urban/vehicles/large_vehicles/mega_hauler_truck
+	name = "mega-hauler truck"
+	icon_state = "longtruck_kellandmining"
+	desc = "Seems to be broken down."
+	icon = 'icons/obj/structures/prop/urban/128x32_vehiclesexpanded.dmi'
+	bound_height = 64
+	bound_width = 128
+	resistance_flags = RESIST_ALL
+	density = TRUE
+
+/obj/structure/prop/urban/vehicles/large_vehicles/mega_hauler_truck/kelland
+	icon_state = "longtruck_kellandmining"
+
+/obj/structure/prop/urban/vehicles/large_vehicles/mega_hauler_truck/red_stripe
+	icon_state = "longtruck_blue_redstripe"
+
+/obj/structure/prop/urban/vehicles/large_vehicles/mega_hauler_truck/blue_stripe
+	icon_state = "longtruck_red_bluestripe"
+
+/obj/structure/prop/urban/vehicles/large_vehicles/mega_hauler_truck/brown
+	icon_state = "longtruck_brown"
+
+/obj/structure/prop/urban/vehicles/large_vehicles/mega_hauler_truck/donk
+	icon_state = "longtruck_donk"
+
+//WY
+/obj/structure/prop/urban/vehicles/large_vehicles/mega_hauler_truck/nt_black
+	name = "Nanotrasen mega-hauler truck"
+	icon_state = "longtruck_nt_black"
+
+/obj/structure/prop/urban/vehicles/large_vehicles/mega_hauler_truck/nt_blue
+	name = "Nanotrasen mega-hauler truck"
```

Comment:
All these ones that do start with a capital, are ones that ACTUALLY need /improper